packages/corvid-cli/src/utils/electron.js

```diff
@@ -144,7 +144,7 @@ function openWindow(windowOptions = {}) {
     height: 960,
     ...windowOptions,
     show: windowOptions.show || disableHeadlessMode,
-    webPreferences: { nodeIntegration: false }
+    webPreferences: { nativeWindowOpen: true }
```

Comment:
was needed in order to allow logging in with Google (there is an open electron issue)